firestore/CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Overview
+
+This describes how Firestore Unity SDK works, and how to develop and test 
+the SDK, targeting desktop/Android/iOS.
+
+# Prerequisites
+
+Building the Unity SDK requires building the underlying C++ SDK. Refer to
+[][this doc] for what the prerequisites are.
+
+On top of above, you also need Unity installed (obviously). If you use an
+apple silicon machine as host, be sure to install the right version of
+Unity!
+
+# Building Firestore Unity SDK
+
+Building Firestore into Unity Packages is a very involved process, mixed with
+multiple build tools working together. Therefore, we will rely on Python scripts
+to automate the process. The scripts live under `$REPO_ROOT/scripts/build_scripts`.
+
+```zsh
+# all scripts are run from the repo root.
+
+# Building for Mac. The build tools will try to find Unity automatically
+python scripts/build_scripts/build_zips.py -platform=macos -targets=auth -targets=firestore -use_boringssl
```

Comment:
Do you have to use boringssl? It generally gives faster build times to use a precompiled openssl, such as the one installed via homebrew.